app/build.gradle

```diff
@@ -25,25 +25,22 @@ if (enable_plugin_kotlin) {
     apply plugin: 'kotlin-kapt'
 }
 
-ext.version_setup_minSdk = 14
-
 android {
-    buildToolsVersion version_setup_buildTools
-    compileSdkVersion version_setup_compileSdk
-    flavorDimensions "default"
+    compileSdkVersion version_compileSdk
+    buildToolsVersion version_buildTools
 
     defaultConfig {
-        minSdkVersion version_setup_minSdk
-        targetSdkVersion version_setup_targetSdk
+        applicationId "io.github.gsantner.memetastic"
+        minSdkVersion version_minSdk
+        targetSdkVersion version_compileSdk
+        versionName "1.5.0"
+        versionCode 50
+
         buildConfigField "boolean", "IS_TEST_BUILD", "false"
         buildConfigField "boolean", "IS_GPLAY_BUILD", "false"
         buildConfigField "String[]", "DETECTED_ANDROID_LOCALES", "${findUsedAndroidLocales()}"
         buildConfigField "String", "GITHASH", "\"${getGitHash()}\""
         resValue "string", "manifest_package_id", "io.github.gsantner.memetastic"
-
-        applicationId "io.github.gsantner.memetastic"
```

Comment:
Yet simple reason why these 3 lines are together, they are things to change on update